src/main/kotlin/org/jitsi/jibri/service/impl/FileRecordingJibriService.kt

```diff
@@ -98,11 +97,16 @@ data class RecordingMetadata(
  */
 class FileRecordingJibriService(
     private val fileRecordingParams: FileRecordingParams,
-    private val jibriSelenium: JibriSelenium = JibriSelenium(),
-    private val capturer: FfmpegCapturer = FfmpegCapturer(),
+    jibriSelenium: JibriSelenium? = null,
+    capturer: FfmpegCapturer? = null,
     private val processFactory: ProcessFactory = ProcessFactory(),
     fileSystem: FileSystem = FileSystems.getDefault()
 ) : StatefulJibriService("File recording") {
+    init {
+        logger.addContext("session_id", fileRecordingParams.sessionId)
```

Comment:
Should this have it's own logger rather than reusing super's? It would also avoid multiple "init" blocks.

Reply:
I don't think we usually use separate loggers for parent classes (just "enclosing" classes) since the classname would end up being the same.